### DISCUSSION TITLE
New Form Builder feedback

### DISCUSSION THREAD

**Discussion by pavlenex** [btcpayserver/btcpayserver#4357] [Ideas / Feature Requests]
In 1.70 we introduced the form builder, which for now allows you to request buyers email or shipping information through point of sale app and payment request. 

Let us know if you have any feedback or feature requests about the form builder.

**Comment by pavlenex** [comment]
Someone suggested that we add a custom text box. I assume this one can be added within the shipping address where buyer can provide additional delivery instructions, but a way more requested use-case is for donations, so that people can leave comments when donating to someone, so it should also be a separate field called `Comment box` or something similar, since we already use comments in invoices, for LNURL we can parse the comments there, though I'm not sure how we could parse comments from both.

**Comment by iBobik** [comment]
Hi, just created my first form.

JSON is ok, but it would help a lot to have a [link to docs](https://docs.btcpayserver.org/Forms/) here.

I also was confused where to create forms, I was looking for it in the apps. If it would be app then it also will be possible to route it to the custom domain.

**Reply by dstrukt** [reply]
To create forms:
Store Settings > Forms

We're currently building out a GUI for (custom) form creation as well as JSON, should be shipped in the coming weeks!

**Reply by dennisreimann** [reply]
Yes, we are working on improving it with #4968.
